tests/_support/Helper/Acceptance/Plugin.php

```diff
@@ -1000,7 +1066,7 @@ public function testRestrictContentDisplaysContent($I, $visibleContent = 'Visibl
 	public function seeProductOutput($I, $productURL, $text = false, $textColor = false, $backgroundColor = false)
 	{
 		// Confirm that the product stylesheet loaded.
-		$I->seeInSource('<link rel="stylesheet" id="convertkit-product-css" href="' . $_ENV['TEST_SITE_WP_URL'] . '/wp-content/plugins/convertkit/resources/frontend/css/product.css');
```

Comment:
Just checking to see if this is intended.

Reply:
Yes; this is renamed to `button.css`, as the same styles are now used for both the Form Trigger and Product blocks. The check on line 1004 is for this renamed stylesheet.